### PR TITLE
Update zh_cn.json

### PR DESCRIPTION
**Pull request by Cactusstudent**
sorry be late, just correct the 1.18 version of this mod, the translation is more complete and greater. I cant make sure all of this is correct, but the quality of this transltion is higher than the original version. I need someone to review this. To make sure most of the translation is correct. And I noticed some compat mods dont work, i go to 1.12.2 and then make these. so this part translation I cant make sure is correct.
And i want ChuijkYahus to review this, because I done the TIC3 with him, except me, he is also familiar with TiC3.

---

(edit on 8/12/23) tbh, I want to translate the guide book, but i dont have any time after the summer holiday, so... the Chinese Transaltion of the books may come or not.

### CODE REVIEW

**Comment by wendall911** [comment]
Thank you! I will wait for a review, then will update the mod. I really appreciate the work on this.

File: src/main/resources/assets/tinkersurvival/lang/zh_cn.json [hunk 1]
```diff
@@ -11,107 +11,107 @@
...
-    "message.wrong_tool2": "嗯... %s ？",
-    "message.tool_broke": "你的 %s 坏了。",
+    "message.wrong_tool": "需要%s。",
+    "message.wrong_tool2": "嗯……%s？",
+    "message.tool_broke": "你的%s坏了。",
     "message.spawn_set": "一点都不困，只是设置了重生点！",
     "effect.tinkersurvival.stop_bleeding": "止血",
     "effect.tinkersurvival.zombie_essence": "僵尸精华",
-    "material.jelledslime": "冰冻史莱姆",
+    "material.jelledslime": "冻结黏液",
```
Comment (ChuijkYahus) [inline src/main/resources/assets/tinkersurvival/lang/zh_cn.json:65]:
```suggestion
    "material.jelledslime": "凝结黏液",
```
maybe "congealed" is better than "frozen"?
may require a check on recipe or interop with other mods(possibly Tough as Nails) in-game

File: src/main/resources/assets/tinkersurvival/lang/zh_cn.json [hunk 2]
```diff
@@ -11,107 +11,107 @@
...
-    "SLOW_DOWN_DIRT_PUNCHING.tooltip": "包括泥土空手挖掘速度也变慢。默认为开启",
-    "ENABLE_SAW.tooltip": "锯子能够制作木板。默认为开启",
-    "FORCE_SAW_FOR_PLANKS.tooltip": "从游戏中去掉木板和木棍的配方，只有锯子才能够制作它们。默认为开启",
+    "balance": "平衡性",
+    "ENABLE_ROCKGEN.tooltip": "岩石堆能生成在地表。默认为true",
+    "ROCKGEN_CHANCE.tooltip": "岩石堆生成在地表的概率。默认为1.0D（=100%）",
+    "ENABLE_ROCK_FROM_DIRT.tooltip": "挖掘泥土能掉落小型岩石。（默认：false)",
+    "ROCK_FROM_DIRT_CHANCE.tooltip": "空手挖掘泥土掉落小型岩石的概率。默认为0.4（=40%）",
+    "FLINT_CHANCE.tooltip": "敲击获得燧石碎片的成功概率。默认为60%-0.6D",
+    "GRASS_FIBER_CHANCE.tooltip": "高草掉落植物纤维的概率。默认为60%-0.6D。使用小刀的概率再多40%。",
```
Comment (ChuijkYahus) [inline src/main/resources/assets/tinkersurvival/lang/zh_cn.json]:
```suggestion
    "GRASS_FIBER_CHANCE.tooltip": "高草丛掉落植物纤维的概率。默认为60%-0.6D。使用小刀的概率再多40%。",
```

File: src/main/resources/assets/tinkersurvival/lang/zh_cn.json [hunk 3]
```diff
@@ -11,107 +11,107 @@
...
+    "block.tinkersurvival.rock_stone_block": "小块岩石",
+    "block.tinkersurvival.stone_loose_rock": "松散的岩石堆（石头）",
+    "block.tinkersurvival.andesite_loose_rock": "松散的岩石堆（安山岩）",
+    "block.tinkersurvival.diorite_loose_rock": "松散的岩石堆（闪长岩）",
+    "block.tinkersurvival.granite_loose_rock": "松散的岩石堆（花岗岩）",
+    "block.tinkersurvival.sandstone_loose_rock": "松散的岩石堆（砂岩）",
+    "block.tinkersurvival.red_sandstone_loose_rock": "松散的岩石堆（红砂岩）",
+    "jei.tinkersurvival.description.rock": "你可以在地上捡到这些岩石，用燧石小刀可以将它切割成燧石碎片",
+    "jei.tinkersurvival.description.plant_fiber": "用小刀斩断草时会掉落植物纤维。植物纤维可以合成用于制作工具的草绳和制作绷带软膏的植物浆。",
+    "jei.tinkersurvival.description.stick": "破坏树叶时会掉落木棍。使用任何小刀都可以增加木棍掉落的概率。",
```
Comment (ChuijkYahus) [inline src/main/resources/assets/tinkersurvival/lang/zh_cn.json]:
```suggestion
    "jei.tinkersurvival.description.stick": "破坏树叶时会掉落木棍。使用任何小刀都可以使木棍掉落的概率翻倍。",
```
original text: `Sticks are dropped from leaves. Using any knife doubles the chance of getting a stick.`
fixes the multiplier part

File: src/main/resources/assets/tinkersurvival/lang/zh_cn.json [hunk 4]
```diff
@@ -11,107 +11,107 @@
...
+    "ENABLE_ROCKGEN.tooltip": "岩石堆能生成在地表。默认为true",
+    "ROCKGEN_CHANCE.tooltip": "岩石堆生成在地表的概率。默认为1.0D（=100%）",
+    "ENABLE_ROCK_FROM_DIRT.tooltip": "挖掘泥土能掉落小型岩石。（默认：false)",
+    "ROCK_FROM_DIRT_CHANCE.tooltip": "空手挖掘泥土掉落小型岩石的概率。默认为0.4（=40%）",
+    "FLINT_CHANCE.tooltip": "敲击获得燧石碎片的成功概率。默认为60%-0.6D",
+    "GRASS_FIBER_CHANCE.tooltip": "高草掉落植物纤维的概率。默认为60%-0.6D。使用小刀的概率再多40%。",
+    "HEAL_RATE.tooltip": "调整绷带的生命值恢复速度。粗制绷带的效果为50%。",
+    "SLOW_DOWN_MULTIPLIER.tooltip": "用于调整错误使用工具时的挖掘速度削减。（默认为1.0)",
+    "features": "游戏特性",
+    "NO_GRIEFING.tooltip": "别让末影人再搞破坏了，什么？！Notch。",
```
Comment (ChuijkYahus) [inline src/main/resources/assets/tinkersurvival/lang/zh_cn.json:92]:
```suggestion
    "NO_GRIEFING.tooltip": "禁止末影人破坏事物，Notch你在做啥。",
```
This change is only a suggestion
I suppose the first half of sentence is an independent statement from the other half, possibly wrong, no need to change that part if finding it strange to read after editing
The second part is changed to a somewhat question(lit. what are you doing Notch.), maybe it fits the original text better

File: src/main/resources/assets/tinkersurvival/lang/zh_cn.json [hunk 5]
```diff
@@ -11,107 +11,107 @@
...
+    "GRIEFING_WHITELIST.tooltip": "OK，我喜欢末影人，如果NO_GRIEFING=true时，它们会捡起这些物品",
+    "NO_SLEEPING.tooltip": "到了晚上，我哭着想跑去睡觉……",
+    "SLOW_DOWN_DIRT_PUNCHING.tooltip": "空手挖掘包含泥土在内的方块的挖掘速度会减慢。默认为true",
+    "ENABLE_SAW.tooltip": "使锯子能够制作木板。默认为true",
+    "FORCE_SAW_FOR_PLANKS.tooltip": "移除原版中木板和木棍和合成配方，且仅能通过锯子制作。默认为true",
```
Comment (ChuijkYahus) [inline src/main/resources/assets/tinkersurvival/lang/zh_cn.json]:
```suggestion
    "FORCE_SAW_FOR_PLANKS.tooltip": "移除原版中木板和木棍的合成配方，且仅能通过锯子制作。默认为true",
```

File: src/main/resources/assets/tinkersurvival/lang/zh_cn.json [hunk 6]
```diff
@@ -11,107 +11,107 @@
...
-    "ARMOR_WHITELIST.tooltip": "仍然能使用的个别盔甲列表。格式 [modid:item]",
-    "ARMOR_TEMP_MODIFIERS.tooltip": "所有有温度调节功能的盔甲列表。格式 [modid:item=modifier]",
+    "MOD_TOOL_WHITELIST.tooltip": "列表中模组的工具将始终有效并且可以制作。而其他模组的工具则会变得像湿面条一样。",
+    "TOOLS_WHITELIST.tooltip": "列表中始终有效的单个工具。格式为tooltype-modid:item",
+    "armor": "盔甲",
+    "MOD_ARMOR_WHITELIST.tooltip": "列表中模组的盔甲将始终有效并且可以制作。未在列表中的模组的盔甲将无法使用并无法制作。",
+    "ARMOR_WHITELIST.tooltip": "列表中始终有效的单个盔甲。格式为modid:item",
+    "ARMOR_TEMP_MODIFIERS.tooltip": "所有有温度调节强化的盔甲列表。格式为modid:item=modifier",
     "INSULATED_MODIFIER.tooltip": "匠魂盔甲羊毛的保温系数值。",
     "CHILLING_MODIFIER.tooltip": "匠魂盔甲冻结史莱姆的降温系数值。",
```
Comment (ChuijkYahus) [inline src/main/resources/assets/tinkersurvival/lang/zh_cn.json]: `冻结史莱姆` should be changed to match its material name above

File: src/main/resources/assets/tinkersurvival/lang/zh_cn.json (hunk 1 above)
Comment (Cactusstudent) [reply src/main/resources/assets/tinkersurvival/lang/zh_cn.json:65]: this material doesnt have any recipe and I even not see it in 1.18 TAN... so i used the translation of 1.12 version.

File: src/main/resources/assets/tinkersurvival/lang/zh_cn.json (hunk 3 above)
Comment (Cactusstudent) [reply src/main/resources/assets/tinkersurvival/lang/zh_cn.json]: it's better, thanks

File: src/main/resources/assets/tinkersurvival/lang/zh_cn.json (hunk 4 above)
Comment (Cactusstudent) [reply src/main/resources/assets/tinkersurvival/lang/zh_cn.json:92]: I have no idea of this, so I just improve the orginal translation. Maybe we should discuss this

File: src/main/resources/assets/tinkersurvival/lang/zh_cn.json (hunk 6 above)
Comment (Cactusstudent) [reply src/main/resources/assets/tinkersurvival/lang/zh_cn.json]: the reason same as jelledslime, not found in 1.18

**Review by ChuijkYahus — APPROVED** [review]

**Comment by Cactusstudent** [comment]
@wendall911 We have done all of these, some keys may not translation 100% sure because some mod not provide this item or material, but we think we did our best. and at last, may I request you to check if this translation have some techical mistake, after you done this, you can merge it. I really appreciate your support of our localization efforts. (and sorry to ping you)

**Pull request merged**